leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -420,6 +423,7 @@ public void onSaveInstanceState(Bundle savedInstanceState) {
     if (poisList != null) {
       savedInstanceState.putParcelableArrayList(POIS_LIST, new ArrayList<>(poisList));
     }
+    savedInstanceState.putBoolean(enableSatelliteView, true);
```

Comment:
You need to use the constant "ENABLE_SATELLITE_VIEW" and the value instead of this.

if (enableSatelliteView != null) {
  savedInstanceState.putBoolean(ENABLE_SATELLITE_VIEW, enableSatelliteView);
}